### ISSUE TITLE
ignored errors still fail the build

### ISSUE THREAD

**Issue by crarchitech** [yargalot/grunt-accessibility#13]
I have 'WCAG2A.Principle2.Guideline2_4.2_4_2.H25.1.NoTitleEl' ignored but the build still breaks.  This is caused by lines 43-45 in the grunt-accessibility.js.

```
      if (msgSplit[0] === 'ERROR' && !options.force) {
        grunt.fail.warn(msgSplit[1] + ': ' + msgSplit[2]);
      }
```

The problem is the ignore logic never gets a chance to run before the build stops.  If this block is moved down to line 56 after the following block, it works properly:

```
      if (ignore) {
        return;
      }
```

Thanks!
